impeller/compiler/shader_lib/impeller/branching.glsl

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef BRANCHING_GLSL_
+#define BRANCHING_GLSL_
+
+#include <impeller/constants.glsl>
+
+vec3 EqualTo3(vec3 x, float y) {
```

Comment:
How about using C style "namespacing" for this library code? For instance, "IPVec3IsEqual"? "IP" the Impeller namespace, "Vec3" the class and then the method name.